### ISSUE TITLE
[Feature Request] Two Column Inventory.

### ISSUE THREAD

**Issue by Zaphero1** [sfall-team/sfall#495]
Sorry if this is a duplicate, I couldn't find an issue for this.

Apparently someone has made their own version of sfall for Fallout: Nevada that improves the inventory UI by adding another column, as shown in Warlockracy's video on the game. 
Does anyone know more about this fork and can this be implemented as a general feature for Fallout 2 and potentially all games on this engine?
![image](https://github.com/sfall-team/sfall/assets/19298996/6a71769f-89c0-4036-8c43-d40e070a401f)

**Comment by Lexx2k** [comment]
Code for that isn't open source, and there's nobody around who can replicate it. So yes it's possible, but only if someone shows up who can do it.

**Comment by phobos2077** [comment]
Yep, this is a highly sought after feature. But as @Lexx2k said, there very few people who work on the engine (sfall or CE). I did some basic research using CE code base to asses how hard it is to do. It's not super hard, especially within CE code, but far from trivial given UI code is difficult even in modern game dev, not to mention engine from mid-1990's.

So if anyone wants to give this a try, you're very welcomed. I was thinking about giving this a try some time when I'm in the mood but only for CE since for me this feature doesn't feel essential tbh, at least when you use InventoryFilter mod (from the same dude as this feature btw) and playing vanilla or mod with not too many new item types.

**Comment by xxTONxx** [comment]
> Apparently someone has made their own version of sfall for Fallout: Nevada that improves the inventory UI

You can download [sfall 5](https://gitflic.ru/project/fakelshub/sfall/release) from the repository.
